### ISSUE TITLE
certificate/key support for auth

### ISSUE THREAD

**Issue by sergey-safarov** [proxytunnel/proxytunnel#51]
I have configured on HTTS server requirement for client certificate for authentication.
Without a client certificate, HTTPS servers refuse connections.
I want to specify certificate and key in `proxytunnel` options but not see such options.
Could you add options for the client certificate key?

I expecting use command like
```
proxytunnel -v -E \
  --client-cert my_cert.pem \
  --client-key my_key.pem \
  -p proxy.example.com:3129 \
  -d ifconfig.me:80
```

**Comment by sigio** [comment]
This application has been abandonware for the last ~10 years.... so If someone builds it, I will happily merge it, but I don't have time (or incentive) to add features.

**Comment by sergey-safarov** [comment]
Thanks for your response.
If nobody interest to add a feature, then you are free to close at any time.

For now, I use `stunnel` and `nc` utilities.
Example: connect to ssh port via HTTPS proxy 
```
ssh -o ServerAliveInterval=3 \
    -o "ProxyCommand=nc --proxy 127.0.0.1:3129 %h %p" \
    -i my_ssh_key.pem centos@example.com
```
On localhost port 3129 started `stunnel` with config
```
client = yes

[squid]
accept = 3129
connect = proxy.example.com:3129
cert = /etc/stunnel/stunnel.pem
```
Cert/key file created using command
```
cat my_cert.key my_cert.pem > /etc/stunnel/stunnel.pem
```

*More info*
stunnel configuration [manual](https://www.digitalocean.com/community/tutorials/how-to-set-up-an-ssl-tunnel-using-stunnel-on-ubuntu).

**Comment by 68420948** [comment]
Implemented with proxytunnel 1.12.0
